internal/hud/view/model_to_view.go

```diff
@@ -0,0 +1,43 @@
+package view
+
+import (
+	"time"
+
+	hudmodel "github.com/windmilleng/tilt/internal/hud/model"
+	tiltmodel "github.com/windmilleng/tilt/internal/model"
+)
+
+func NewView(m hudmodel.Model) View {
```

Comment:
spidey sense says that this should be a method ON a model (`m.ToView` or `m.NewView` or whatever), but i don't care too much 🤷‍♀️

Reply:
That's what I had originally, but then I figured maybe it's better to not give `Model` a dependency on `View`. If, e.g., at some point we have some other kind of view for a different UI, then it's weird to just have the one "NewView" instance method. That's so hypothetical and cheap to change anyway that it doesn't really matter, but I didn't have any good arguments in the other direction, either.